README.md

```diff
@@ -1,4 +1,4 @@
-# GitLab Pipeline for ORT
+# GitLab Job for ORT
```

Comment:
Have you considered "GitLab Job Template for ORT" ?

Reply:
GitLab itself has the concept of [job templates](https://docs.gitlab.com/ee/ci/examples/#cicd-templates) and what is in this repository is slightly different.